client/src/components/TrackList.vue

```diff
@@ -55,7 +55,7 @@ export default defineComponent({
 
   components: { TrackItem },
 
-  setup(props) {
+  setup(props, { emit }) {
```

Comment:
Unused?

Reply:
yup thanks, I was going to place the Home/End call in the `TrackList` until I decided it made more sense to update how `TrackItem.vue` uses the v-mousetrap to prevent all the excess calls and binding on each item.